kak-lsp.toml

```diff
@@ -6,163 +6,184 @@ verbosity = 2
 # set to 0 to disable
 timeout = 1800 # seconds = 30 minutes
 
-[language.bash]
+# This section overrides language IDs.
+# By default, kak-lsp use filetypes for the IDs.
+# See https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocumentItem
+[language_ids]
```

Comment:
Hypothetically, if a user wanted to use clangd for the `c` and `c++` filetype but with different parameters each, that'd be possible now.
They can define `language_server.clangd-for-c` and `language_server.clangd-for-cpp`.
Nice.